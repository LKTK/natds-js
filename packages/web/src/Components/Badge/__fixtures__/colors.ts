import { BadgeColor } from "../Badge.props";

export const colors: Record<BadgeColor, BadgeColor> = {
  default: "default",
  error: "error",
  primary: "primary",
  secondary: "secondary",
};

export default colors;
