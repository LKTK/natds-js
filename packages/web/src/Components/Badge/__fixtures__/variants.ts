import { BadgeVariant } from "../Badge.props";

export const variants: Record<BadgeVariant, BadgeVariant> = {
  dot: "dot",
  standard: "standard",
};

export default variants;
