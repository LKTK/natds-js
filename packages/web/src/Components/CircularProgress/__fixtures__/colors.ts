import { ICircularProgressProps } from "../CircularProgress.props";

export const colors : Record<ICircularProgressProps["color"], ICircularProgressProps["color"]> = {
  primary: "primary",
  secondary: "secondary",
  inherit: "inherit",
};

export default colors;
