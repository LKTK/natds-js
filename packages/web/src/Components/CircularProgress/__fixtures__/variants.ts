import { ICircularProgressProps } from "../CircularProgress.props";

export const variants : Record<ICircularProgressProps["variant"], ICircularProgressProps["variant"]> = {
  determinate: "determinate",
  indeterminate: "indeterminate",
  static: "static",
};

export default variants;
