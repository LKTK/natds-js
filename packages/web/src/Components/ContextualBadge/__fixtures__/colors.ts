import { IContextualBadgeColors } from "../ContextualBadge.props";

export const colors : Record<IContextualBadgeColors, IContextualBadgeColors> = {
  dark: "dark",
  error: "error",
  light: "light",
  info: "info",
  primary: "primary",
  secondary: "secondary",
  success: "success",
  warning: "warning",
};

export default colors;
