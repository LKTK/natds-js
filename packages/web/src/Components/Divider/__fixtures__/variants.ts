import { IDividerProps } from "../Divider.props";

export const variants: Record<IDividerProps["variant"], IDividerProps["variant"]> = {
  fullWidth: "fullWidth",
  inset: "inset",
  middle: "middle",
};

export default variants;
