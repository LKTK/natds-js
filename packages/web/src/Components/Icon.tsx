import React, { FunctionComponent } from 'react';
import { IThemeWeb } from 'Themes';

import { withTheme } from '@material-ui/styles';
import '@naturacosmeticos/natds-icons/dist/natds-icons.css';

interface IIconProps {
  name: string;
  theme?: IThemeWeb | unknown;
}

const Icon: FunctionComponent<IIconProps> = (props: IIconProps) => {
  const { name } = props;

  return (
    <i className={`natds-icons natds-icons-${name} natds-icons-1x`} />
  );
};

export default withTheme(Icon);
