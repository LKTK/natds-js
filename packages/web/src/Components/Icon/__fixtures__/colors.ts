import { IconColor } from "../Icon.props";

/**
 * @todo add support to color tokens from Natura & Co. themes
 */
export const colors : Record<IconColor, IconColor> = {
  default: "default",
  inherit: "inherit",
  primary: "primary",
  secondary: "secondary",
};
