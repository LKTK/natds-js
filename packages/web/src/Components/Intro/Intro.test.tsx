import * as React from "react";
import * as TestRenderer from "react-test-renderer";

import Icon from "../Icon";
import { Intro } from "./index";
import { TypographyColor, TypographyVariant } from "../Typography";
import { Provider } from "../../Provider";
import { themes } from "../../Themes";
import { variants } from "./__fixtures__/variants";
import { colors } from "./__fixtures__/colors";

const theme = themes.natura.light;

const TitleWithIcon = () => <>
  <Icon name="filled-action-add" size="tiny" />
    title
</>;

describe("Intro component", () => {

  let testRenderer : TestRenderer.ReactTestRenderer = {} as TestRenderer.ReactTestRenderer;

  it("should match snapshot - Intro with title and without detail", () => {
    testRenderer = TestRenderer.create(<Provider theme={theme}><Intro title="title" /></Provider>);

    expect(testRenderer).toMatchSnapshot();
  });

  it("should match snapshot - Intro with title and detail", () => {
    testRenderer = TestRenderer.create(<Provider theme={theme}><Intro title="title" detail="detail" /></Provider>);

    expect(testRenderer).toMatchSnapshot();
  });

  it("should match snapshot - Intro with title and icon", () => {
    testRenderer = TestRenderer.create(<Provider theme={theme}><Intro title={<TitleWithIcon />} /></Provider>);

    expect(testRenderer).toMatchSnapshot();
  });

  Object.keys(variants).forEach((variant: string) => {
    it(`should match snapshot - Any valid Intro with ${variant} titleVariant`, () => {
      testRenderer = TestRenderer.create(<Provider theme={theme}><Intro titleVariant={variant as TypographyVariant} title="title" /></Provider>);

      expect(testRenderer).toMatchSnapshot();
    });

    it(`should match snapshot - Any valid Intro with ${variant} detailVariant`, () => {
      testRenderer = TestRenderer.create(<Provider theme={theme}><Intro detail="detail"
        detailVariant={variant as TypographyVariant}
        title="title"/></Provider>);

      expect(testRenderer).toMatchSnapshot();
    });
  });

  Object.keys(colors).forEach((color: string) => {
    it(`should match snapshot - Any valid Intro with ${color} titleColor`, () => {
      testRenderer = TestRenderer.create(<Provider theme={theme}><Intro titleColor={color as TypographyColor} title="title" /></Provider>);

      expect(testRenderer).toMatchSnapshot();
    });

    it(`should match snapshot - Any valid Intro with ${color} detailColor`, () => {
      testRenderer = TestRenderer.create(<Provider theme={theme}><Intro detail="detail" detailColor={color as TypographyColor} title="title" /></Provider>);

      expect(testRenderer).toMatchSnapshot();
    });
  });
});
