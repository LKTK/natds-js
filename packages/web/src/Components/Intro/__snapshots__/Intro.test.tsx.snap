// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Intro component should match snapshot - Any valid Intro with body1 detailVariant 1`] = `
<section
  className="NatDSIntro-root-83"
>
  <h3
    className="MuiTypography-root NatDSIntro-introTitle-85 NatDSIntro-introTitle-86 MuiTypography-subtitle1 MuiTypography-colorTextPrimary"
  >
    title
  </h3>
  <p
    className="MuiTypography-root NatDSIntro-introDetail-84 MuiTypography-body1 MuiTypography-colorTextPrimary"
  >
    detail
  </p>
</section>
`;

exports[`Intro component should match snapshot - Any valid Intro with body1 titleVariant 1`] = `
<section
  className="NatDSIntro-root-79"
>
  <h3
    className="MuiTypography-root NatDSIntro-introTitle-81 NatDSIntro-introTitle-82 MuiTypography-body1 MuiTypography-colorTextPrimary"
  >
    title
  </h3>
</section>
`;

exports[`Intro component should match snapshot - Any valid Intro with body2 detailVariant 1`] = `
<section
  className="NatDSIntro-root-91"
>
  <h3
    className="MuiTypography-root NatDSIntro-introTitle-93 NatDSIntro-introTitle-94 MuiTypography-subtitle1 MuiTypography-colorTextPrimary"
  >
    title
  </h3>
  <p
    className="MuiTypography-root NatDSIntro-introDetail-92 MuiTypography-body2 MuiTypography-colorTextPrimary"
  >
    detail
  </p>
</section>
`;

exports[`Intro component should match snapshot - Any valid Intro with body2 titleVariant 1`] = `
<section
  className="NatDSIntro-root-87"
>
  <h3
    className="MuiTypography-root NatDSIntro-introTitle-89 NatDSIntro-introTitle-90 MuiTypography-body2 MuiTypography-colorTextPrimary"
  >
    title
  </h3>
</section>
`;

exports[`Intro component should match snapshot - Any valid Intro with button detailVariant 1`] = `
<section
  className="NatDSIntro-root-107"
>
  <h3
    className="MuiTypography-root NatDSIntro-introTitle-109 NatDSIntro-introTitle-110 MuiTypography-subtitle1 MuiTypography-colorTextPrimary"
  >
    title
  </h3>
  <p
    className="MuiTypography-root NatDSIntro-introDetail-108 MuiTypography-button MuiTypography-colorTextPrimary"
  >
    detail
  </p>
</section>
`;

exports[`Intro component should match snapshot - Any valid Intro with button titleVariant 1`] = `
<section
  className="NatDSIntro-root-103"
>
  <h3
    className="MuiTypography-root NatDSIntro-introTitle-105 NatDSIntro-introTitle-106 MuiTypography-button MuiTypography-colorTextPrimary"
  >
    title
  </h3>
</section>
`;

exports[`Intro component should match snapshot - Any valid Intro with caption detailVariant 1`] = `
<section
  className="NatDSIntro-root-99"
>
  <h3
    className="MuiTypography-root NatDSIntro-introTitle-101 NatDSIntro-introTitle-102 MuiTypography-subtitle1 MuiTypography-colorTextPrimary"
  >
    title
  </h3>
  <p
    className="MuiTypography-root NatDSIntro-introDetail-100 MuiTypography-caption MuiTypography-colorTextPrimary"
  >
    detail
  </p>
</section>
`;

exports[`Intro component should match snapshot - Any valid Intro with caption titleVariant 1`] = `
<section
  className="NatDSIntro-root-95"
>
  <h3
    className="MuiTypography-root NatDSIntro-introTitle-97 NatDSIntro-introTitle-98 MuiTypography-caption MuiTypography-colorTextPrimary"
  >
    title
  </h3>
</section>
`;

exports[`Intro component should match snapshot - Any valid Intro with error detailColor 1`] = `
<section
  className="NatDSIntro-root-179"
>
  <h3
    className="MuiTypography-root NatDSIntro-introTitle-181 NatDSIntro-introTitle-182 MuiTypography-subtitle1 MuiTypography-colorTextPrimary"
  >
    title
  </h3>
  <p
    className="MuiTypography-root NatDSIntro-introDetail-180 MuiTypography-body2 MuiTypography-colorError"
  >
    detail
  </p>
</section>
`;

exports[`Intro component should match snapshot - Any valid Intro with error titleColor 1`] = `
<section
  className="NatDSIntro-root-175"
>
  <h3
    className="MuiTypography-root NatDSIntro-introTitle-177 NatDSIntro-introTitle-178 MuiTypography-subtitle1 MuiTypography-colorError"
  >
    title
  </h3>
</section>
`;

exports[`Intro component should match snapshot - Any valid Intro with h1 detailVariant 1`] = `
<section
  className="NatDSIntro-root-19"
>
  <h3
    className="MuiTypography-root NatDSIntro-introTitle-21 NatDSIntro-introTitle-22 MuiTypography-subtitle1 MuiTypography-colorTextPrimary"
  >
    title
  </h3>
  <p
    className="MuiTypography-root NatDSIntro-introDetail-20 MuiTypography-h1 MuiTypography-colorTextPrimary"
  >
    detail
  </p>
</section>
`;

exports[`Intro component should match snapshot - Any valid Intro with h1 titleVariant 1`] = `
<section
  className="NatDSIntro-root-15"
>
  <h3
    className="MuiTypography-root NatDSIntro-introTitle-17 NatDSIntro-introTitle-18 MuiTypography-h1 MuiTypography-colorTextPrimary"
  >
    title
  </h3>
</section>
`;

exports[`Intro component should match snapshot - Any valid Intro with h2 detailVariant 1`] = `
<section
  className="NatDSIntro-root-27"
>
  <h3
    className="MuiTypography-root NatDSIntro-introTitle-29 NatDSIntro-introTitle-30 MuiTypography-subtitle1 MuiTypography-colorTextPrimary"
  >
    title
  </h3>
  <p
    className="MuiTypography-root NatDSIntro-introDetail-28 MuiTypography-h2 MuiTypography-colorTextPrimary"
  >
    detail
  </p>
</section>
`;

exports[`Intro component should match snapshot - Any valid Intro with h2 titleVariant 1`] = `
<section
  className="NatDSIntro-root-23"
>
  <h3
    className="MuiTypography-root NatDSIntro-introTitle-25 NatDSIntro-introTitle-26 MuiTypography-h2 MuiTypography-colorTextPrimary"
  >
    title
  </h3>
</section>
`;

exports[`Intro component should match snapshot - Any valid Intro with h3 detailVariant 1`] = `
<section
  className="NatDSIntro-root-35"
>
  <h3
    className="MuiTypography-root NatDSIntro-introTitle-37 NatDSIntro-introTitle-38 MuiTypography-subtitle1 MuiTypography-colorTextPrimary"
  >
    title
  </h3>
  <p
    className="MuiTypography-root NatDSIntro-introDetail-36 MuiTypography-h3 MuiTypography-colorTextPrimary"
  >
    detail
  </p>
</section>
`;

exports[`Intro component should match snapshot - Any valid Intro with h3 titleVariant 1`] = `
<section
  className="NatDSIntro-root-31"
>
  <h3
    className="MuiTypography-root NatDSIntro-introTitle-33 NatDSIntro-introTitle-34 MuiTypography-h3 MuiTypography-colorTextPrimary"
  >
    title
  </h3>
</section>
`;

exports[`Intro component should match snapshot - Any valid Intro with h4 detailVariant 1`] = `
<section
  className="NatDSIntro-root-43"
>
  <h3
    className="MuiTypography-root NatDSIntro-introTitle-45 NatDSIntro-introTitle-46 MuiTypography-subtitle1 MuiTypography-colorTextPrimary"
  >
    title
  </h3>
  <p
    className="MuiTypography-root NatDSIntro-introDetail-44 MuiTypography-h4 MuiTypography-colorTextPrimary"
  >
    detail
  </p>
</section>
`;

exports[`Intro component should match snapshot - Any valid Intro with h4 titleVariant 1`] = `
<section
  className="NatDSIntro-root-39"
>
  <h3
    className="MuiTypography-root NatDSIntro-introTitle-41 NatDSIntro-introTitle-42 MuiTypography-h4 MuiTypography-colorTextPrimary"
  >
    title
  </h3>
</section>
`;

exports[`Intro component should match snapshot - Any valid Intro with h5 detailVariant 1`] = `
<section
  className="NatDSIntro-root-51"
>
  <h3
    className="MuiTypography-root NatDSIntro-introTitle-53 NatDSIntro-introTitle-54 MuiTypography-subtitle1 MuiTypography-colorTextPrimary"
  >
    title
  </h3>
  <p
    className="MuiTypography-root NatDSIntro-introDetail-52 MuiTypography-h5 MuiTypography-colorTextPrimary"
  >
    detail
  </p>
</section>
`;

exports[`Intro component should match snapshot - Any valid Intro with h5 titleVariant 1`] = `
<section
  className="NatDSIntro-root-47"
>
  <h3
    className="MuiTypography-root NatDSIntro-introTitle-49 NatDSIntro-introTitle-50 MuiTypography-h5 MuiTypography-colorTextPrimary"
  >
    title
  </h3>
</section>
`;

exports[`Intro component should match snapshot - Any valid Intro with h6 detailVariant 1`] = `
<section
  className="NatDSIntro-root-59"
>
  <h3
    className="MuiTypography-root NatDSIntro-introTitle-61 NatDSIntro-introTitle-62 MuiTypography-subtitle1 MuiTypography-colorTextPrimary"
  >
    title
  </h3>
  <p
    className="MuiTypography-root NatDSIntro-introDetail-60 MuiTypography-h6 MuiTypography-colorTextPrimary"
  >
    detail
  </p>
</section>
`;

exports[`Intro component should match snapshot - Any valid Intro with h6 titleVariant 1`] = `
<section
  className="NatDSIntro-root-55"
>
  <h3
    className="MuiTypography-root NatDSIntro-introTitle-57 NatDSIntro-introTitle-58 MuiTypography-h6 MuiTypography-colorTextPrimary"
  >
    title
  </h3>
</section>
`;

exports[`Intro component should match snapshot - Any valid Intro with inherit detailColor 1`] = `
<section
  className="NatDSIntro-root-139"
>
  <h3
    className="MuiTypography-root NatDSIntro-introTitle-141 NatDSIntro-introTitle-142 MuiTypography-subtitle1 MuiTypography-colorTextPrimary"
  >
    title
  </h3>
  <p
    className="MuiTypography-root NatDSIntro-introDetail-140 MuiTypography-body2 MuiTypography-colorInherit"
  >
    detail
  </p>
</section>
`;

exports[`Intro component should match snapshot - Any valid Intro with inherit detailVariant 1`] = `
<section
  className="NatDSIntro-root-123"
>
  <h3
    className="MuiTypography-root NatDSIntro-introTitle-125 NatDSIntro-introTitle-126 MuiTypography-subtitle1 MuiTypography-colorTextPrimary"
  >
    title
  </h3>
  <p
    className="MuiTypography-root NatDSIntro-introDetail-124 MuiTypography-colorTextPrimary"
  >
    detail
  </p>
</section>
`;

exports[`Intro component should match snapshot - Any valid Intro with inherit titleColor 1`] = `
<section
  className="NatDSIntro-root-135"
>
  <h3
    className="MuiTypography-root NatDSIntro-introTitle-137 NatDSIntro-introTitle-138 MuiTypography-subtitle1 MuiTypography-colorInherit"
  >
    title
  </h3>
</section>
`;

exports[`Intro component should match snapshot - Any valid Intro with inherit titleVariant 1`] = `
<section
  className="NatDSIntro-root-119"
>
  <h3
    className="MuiTypography-root NatDSIntro-introTitle-121 NatDSIntro-introTitle-122 MuiTypography-colorTextPrimary"
  >
    title
  </h3>
</section>
`;

exports[`Intro component should match snapshot - Any valid Intro with initial detailColor 1`] = `
<section
  className="NatDSIntro-root-131"
>
  <h3
    className="MuiTypography-root NatDSIntro-introTitle-133 NatDSIntro-introTitle-134 MuiTypography-subtitle1 MuiTypography-colorTextPrimary"
  >
    title
  </h3>
  <p
    className="MuiTypography-root NatDSIntro-introDetail-132 MuiTypography-body2"
  >
    detail
  </p>
</section>
`;

exports[`Intro component should match snapshot - Any valid Intro with initial titleColor 1`] = `
<section
  className="NatDSIntro-root-127"
>
  <h3
    className="MuiTypography-root NatDSIntro-introTitle-129 NatDSIntro-introTitle-130 MuiTypography-subtitle1"
  >
    title
  </h3>
</section>
`;

exports[`Intro component should match snapshot - Any valid Intro with overline detailVariant 1`] = `
<section
  className="NatDSIntro-root-115"
>
  <h3
    className="MuiTypography-root NatDSIntro-introTitle-117 NatDSIntro-introTitle-118 MuiTypography-subtitle1 MuiTypography-colorTextPrimary"
  >
    title
  </h3>
  <p
    className="MuiTypography-root NatDSIntro-introDetail-116 MuiTypography-overline MuiTypography-colorTextPrimary"
  >
    detail
  </p>
</section>
`;

exports[`Intro component should match snapshot - Any valid Intro with overline titleVariant 1`] = `
<section
  className="NatDSIntro-root-111"
>
  <h3
    className="MuiTypography-root NatDSIntro-introTitle-113 NatDSIntro-introTitle-114 MuiTypography-overline MuiTypography-colorTextPrimary"
  >
    title
  </h3>
</section>
`;

exports[`Intro component should match snapshot - Any valid Intro with primary detailColor 1`] = `
<section
  className="NatDSIntro-root-147"
>
  <h3
    className="MuiTypography-root NatDSIntro-introTitle-149 NatDSIntro-introTitle-150 MuiTypography-subtitle1 MuiTypography-colorTextPrimary"
  >
    title
  </h3>
  <p
    className="MuiTypography-root NatDSIntro-introDetail-148 MuiTypography-body2 MuiTypography-colorPrimary"
  >
    detail
  </p>
</section>
`;

exports[`Intro component should match snapshot - Any valid Intro with primary titleColor 1`] = `
<section
  className="NatDSIntro-root-143"
>
  <h3
    className="MuiTypography-root NatDSIntro-introTitle-145 NatDSIntro-introTitle-146 MuiTypography-subtitle1 MuiTypography-colorPrimary"
  >
    title
  </h3>
</section>
`;

exports[`Intro component should match snapshot - Any valid Intro with secondary detailColor 1`] = `
<section
  className="NatDSIntro-root-155"
>
  <h3
    className="MuiTypography-root NatDSIntro-introTitle-157 NatDSIntro-introTitle-158 MuiTypography-subtitle1 MuiTypography-colorTextPrimary"
  >
    title
  </h3>
  <p
    className="MuiTypography-root NatDSIntro-introDetail-156 MuiTypography-body2 MuiTypography-colorSecondary"
  >
    detail
  </p>
</section>
`;

exports[`Intro component should match snapshot - Any valid Intro with secondary titleColor 1`] = `
<section
  className="NatDSIntro-root-151"
>
  <h3
    className="MuiTypography-root NatDSIntro-introTitle-153 NatDSIntro-introTitle-154 MuiTypography-subtitle1 MuiTypography-colorSecondary"
  >
    title
  </h3>
</section>
`;

exports[`Intro component should match snapshot - Any valid Intro with subtitle1 detailVariant 1`] = `
<section
  className="NatDSIntro-root-67"
>
  <h3
    className="MuiTypography-root NatDSIntro-introTitle-69 NatDSIntro-introTitle-70 MuiTypography-subtitle1 MuiTypography-colorTextPrimary"
  >
    title
  </h3>
  <p
    className="MuiTypography-root NatDSIntro-introDetail-68 MuiTypography-subtitle1 MuiTypography-colorTextPrimary"
  >
    detail
  </p>
</section>
`;

exports[`Intro component should match snapshot - Any valid Intro with subtitle1 titleVariant 1`] = `
<section
  className="NatDSIntro-root-63"
>
  <h3
    className="MuiTypography-root NatDSIntro-introTitle-65 NatDSIntro-introTitle-66 MuiTypography-subtitle1 MuiTypography-colorTextPrimary"
  >
    title
  </h3>
</section>
`;

exports[`Intro component should match snapshot - Any valid Intro with subtitle2 detailVariant 1`] = `
<section
  className="NatDSIntro-root-75"
>
  <h3
    className="MuiTypography-root NatDSIntro-introTitle-77 NatDSIntro-introTitle-78 MuiTypography-subtitle1 MuiTypography-colorTextPrimary"
  >
    title
  </h3>
  <p
    className="MuiTypography-root NatDSIntro-introDetail-76 MuiTypography-subtitle2 MuiTypography-colorTextPrimary"
  >
    detail
  </p>
</section>
`;

exports[`Intro component should match snapshot - Any valid Intro with subtitle2 titleVariant 1`] = `
<section
  className="NatDSIntro-root-71"
>
  <h3
    className="MuiTypography-root NatDSIntro-introTitle-73 NatDSIntro-introTitle-74 MuiTypography-subtitle2 MuiTypography-colorTextPrimary"
  >
    title
  </h3>
</section>
`;

exports[`Intro component should match snapshot - Any valid Intro with textPrimary detailColor 1`] = `
<section
  className="NatDSIntro-root-163"
>
  <h3
    className="MuiTypography-root NatDSIntro-introTitle-165 NatDSIntro-introTitle-166 MuiTypography-subtitle1 MuiTypography-colorTextPrimary"
  >
    title
  </h3>
  <p
    className="MuiTypography-root NatDSIntro-introDetail-164 MuiTypography-body2 MuiTypography-colorTextPrimary"
  >
    detail
  </p>
</section>
`;

exports[`Intro component should match snapshot - Any valid Intro with textPrimary titleColor 1`] = `
<section
  className="NatDSIntro-root-159"
>
  <h3
    className="MuiTypography-root NatDSIntro-introTitle-161 NatDSIntro-introTitle-162 MuiTypography-subtitle1 MuiTypography-colorTextPrimary"
  >
    title
  </h3>
</section>
`;

exports[`Intro component should match snapshot - Any valid Intro with textSecondary detailColor 1`] = `
<section
  className="NatDSIntro-root-171"
>
  <h3
    className="MuiTypography-root NatDSIntro-introTitle-173 NatDSIntro-introTitle-174 MuiTypography-subtitle1 MuiTypography-colorTextPrimary"
  >
    title
  </h3>
  <p
    className="MuiTypography-root NatDSIntro-introDetail-172 MuiTypography-body2 MuiTypography-colorTextSecondary"
  >
    detail
  </p>
</section>
`;

exports[`Intro component should match snapshot - Any valid Intro with textSecondary titleColor 1`] = `
<section
  className="NatDSIntro-root-167"
>
  <h3
    className="MuiTypography-root NatDSIntro-introTitle-169 NatDSIntro-introTitle-170 MuiTypography-subtitle1 MuiTypography-colorTextSecondary"
  >
    title
  </h3>
</section>
`;

exports[`Intro component should match snapshot - Intro with title and detail 1`] = `
<section
  className="NatDSIntro-root-5"
>
  <h3
    className="MuiTypography-root NatDSIntro-introTitle-7 NatDSIntro-introTitle-8 MuiTypography-subtitle1 MuiTypography-colorTextPrimary"
  >
    title
  </h3>
  <p
    className="MuiTypography-root NatDSIntro-introDetail-6 MuiTypography-body2 MuiTypography-colorTextPrimary"
  >
    detail
  </p>
</section>
`;

exports[`Intro component should match snapshot - Intro with title and icon 1`] = `
<section
  className="NatDSIntro-root-9"
>
  <h3
    className="MuiTypography-root NatDSIntro-introTitle-11 NatDSIntro-introTitle-12 MuiTypography-subtitle1 MuiTypography-colorTextPrimary"
  >
    <i
      aria-hidden={true}
      className="material-icons MuiIcon-root natds-icons natds-icons-filled-action-add NatDSIcon-root-13 NatDSIcon-root-14"
    >
      filled-action-add
    </i>
    title
  </h3>
</section>
`;

exports[`Intro component should match snapshot - Intro with title and without detail 1`] = `
<section
  className="NatDSIntro-root-1"
>
  <h3
    className="MuiTypography-root NatDSIntro-introTitle-3 NatDSIntro-introTitle-4 MuiTypography-subtitle1 MuiTypography-colorTextPrimary"
  >
    title
  </h3>
</section>
`;
